cratedb_retention/setup/schema.sql

```diff
@@ -26,8 +28,7 @@ CREATE TABLE IF NOT EXISTS {policy_table.fullname} (
     "reallocation_attribute_value" TEXT,        -- Value of the node-specific custom attribute.
 
     -- Targeting a repository.
-    "target_repository_name" TEXT,              -- The name of a repository created with `CREATE REPOSITORY ...`.
+    "target_repository_name" TEXT               -- The name of a repository created with `CREATE REPOSITORY ...`.
 
-    PRIMARY KEY ("strategy", "table_schema", "table_name")
```

Comment:
The idea behind that primary key was to prevent duplicate strategies on the same table. Too bad we don't have `UNIQUE` constraints in CrateDB.
Is there elsewhere in the code a check to prevent duplicates (i.e., for the same table, one entry with `DELETE` and 3 days retention, and another with `DELETE` and 5 days retention on the same table)?

Reply:
Thanks for spotting. I haven't had the chance to outline that regression yet, but I've now created GH-26 to track it.